chia/rpc/full_node_rpc_api.py

```diff
@@ -863,7 +863,7 @@ async def create_block_generator(self, _: dict[str, Any]) -> EndpointResult:
             start_time = time.monotonic()
 
             try:
-                maybe_gen = await self.service.mempool_manager.create_block_generator(curr_l_tb.header_hash)
+                maybe_gen = await self.service.mempool_manager.create_block_generator2(curr_l_tb.header_hash)
```

Comment:
Should this check for `original_block_creation` config to know what version to call?

Reply:
my view of this is RPC is that its main purpose is to test the block creation in the mempool. I think this is the function that warrants testing now